packages/apollo-link-retry/package.json

```diff
@@ -56,7 +56,8 @@
     "ts-jest": "21.2.4",
     "tslint": "5.8.0",
     "typescript": "2.6.2",
-    "uglify-js": "3.2.1"
+    "uglify-js": "3.2.1",
+    "wait-for-observables": "^1.0.3"
```

Comment:
Built just for this :P https://github.com/convoyinc/wait-for-observables - def cleans up the tests

Reply:
I will be using this SO much 😍